grafana-formula/metadata/form.yml

```diff
@@ -90,3 +90,15 @@ grafana:
       $name: Kubernetes namespaces dashboard
       $help: Add dashboard for monitoring Kubernetes namespaces
       $default: False
+
+    add_ha_dashboards:
+      $type: boolean
+      $name: HA pacemaker cluster dashboards
+      $help: Add dashboard for monitoring HA pacemaker clusters
+      $default: False
+
+    add_sap_dashboards:
```

Comment:
I would suggest to follow @stefanotorresi recommendation from above and split it up in two states? Then I would suggest to write both as singular as well, e.g. `add_sap_hana_dashboard` and `add_sap_netweaver_dashboard`.

Reply:
@renner the packages use the plural `dashboards` deliberately because we might provide multiple dashboards within one package.

Reply:
That's fine, feel free to keep the plural here as well in case it makes sense, i.e. more than one dashboard is added with a state.

Reply:
yes is something on my todo list will push a commit on this